malsub/service/joesandbox.py

```diff
@@ -52,19 +62,17 @@ def report_file(self, hash: Hash):
     def submit_file(self, file: File):
         self.api_subf.data = {
             "accept-tac": "1" if self._accept_tac else "0",
-            **self.get_apikey()
-        }
-        self.api_subf.file = {
-            "sample": file.fd()
+            **self.get_apikey(),
```

Comment:
Why is the comma needed here and below?

Reply:
Black will add a trailing comma on each multi-line data structure definition (like this one, and the ones below). It also according to PEP8 - https://www.python.org/dev/peps/pep-0008/#when-to-use-trailing-commas
It has some good reasoning why to do it - to have a clearer diff, easily add more members to the data-structure, and more.
I prefer to leave it like that if you don't mind. If you do, I can find a way in `black` configs to override it.

Reply:
Sounds good to me then. I was unaware that there is guidance to use the trailing commas.